osa/scripts/reprocess_longterm.py

```diff
@@ -31,7 +31,7 @@ def run_longterm(date: str, prod_id: str, new_prod_id: str, log_dir: Path):
         Path to the directory where job logs will be stored.
     """
     dl1_dir = ANALYSIS_PATH / "DL1" / date / prod_id / "tailcut84"
```

Comment:
here the input dir would be no longer `DL1/YYYYMMDD/v0.10/tailcut84/` but `DL1/YYYYMMDD/v0.10/tailcut84/datacheck/` in the new scheme